launch/tier4_control_launch/config/trajectory_follower/lateral_controller.param.yaml

```diff
@@ -40,6 +40,7 @@
     mpc_acceleration_limit: 2.0                  # limit on the vehicle's acceleration
     mpc_velocity_time_constant: 0.3              # time constant used for velocity smoothing
     mpc_min_prediction_length: 5.0               # minimum prediction length
+    mpc_min_vel: 1.0                             # minimum velocity for relative time
```

Comment:
Please check if the control works well for slow-speed products. If not, set this value to 0.1 as previous.

Reply:
remove this param in https://github.com/autowarefoundation/autoware.universe/pull/2447/commits/875a62a1416104f5d2389fac3ca38b4cc889f841, add flag in https://github.com/autowarefoundation/autoware.universe/pull/2447/commits/69e2c0fbf18d55b28add9068c0e1e9e4c2ac65ff.
same to before this PR he min vel in the middle of trajectory is 0.1m/s as a hard code, and 10.0m/s only for exteded points.